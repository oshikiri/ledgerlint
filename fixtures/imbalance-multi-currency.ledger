2020-03-01 ! at super market
  Expences:Food  1000 USD
  Expences:Household essentials  200 JPY
  Assets:Cash  -2000 JPY
