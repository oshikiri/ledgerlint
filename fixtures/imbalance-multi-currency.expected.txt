fixtures/imbalance-multi-currency.ledger:1 imbalanced transaction, (total amount) = 1000 JPY - 1800 USD
